### DISCUSSION TITLE
[design/general] Port TGs wound system.

### DISCUSSION THREAD

**Discussion by hal9000PR** [ParadiseSS13/Paradise#17063] [Documents]
## The Proposal.

Replace the current implementation of internal damage (IB, broken bones) with the TG wound system.

## What does this entail? 

In summary, it adds 4 types of effect, slash, puncture, burn and bone. Each has 3 levels of damage acting as "tiers" the worse the injury the more devastating it is and the harder it is to fix. 

**all** of these are fixable without surgery on TG, however I am not sure if this is ideal for para. I propose that the tier 1 and 2 are fixable without surgery, at some cost, whether it being medical supplies or just time. fixing these should make you vulnerable for a period, allowing people to capitalise on your injury.

SURGERY STILL IS USED TO FIX THESE, THERE ARE JUST "GHETTO" METHODS, THAT ARE SLOWER, MORE EXPENSIVE, BUT CAN BE PERFORMED ON ONES SELF.

all wounds increase the chance of a limb being removed.

**Slash:** 
cause: sharp melee weapons, EG. knives, eswords, fireaxes.
effect: rapid bleeding, that slows over time
fixed by: sutures and clotting medicines
higher levels: increase in bleeding, clotting slowed

**Puncture:**
cause: bullets. 
effect: bleeding, slower than a slash wound, does not clot on its own.
fixed by: sutures and clotting medicines. 
higher levels: increased bleeding.

**Bone:**
cause: brute impacts, harmbatons, toolbox strikes, etc.
effect: similar to current broken bones, starts with displaced joints which only affect arms/legs. cause slowdown/drop items. 
fixed by: applying bone gel, splints, bone tape, re-locating joints (can be done in multiple ways, getting help or a small chance of it being fixed when doorcrushing yourself)
higher levels: can affect all limbs, cause organ damage.

**Burn:**
cause: lasers, welders, other burn based attacks.
effect: limb slowly gets infected, and takes bonus damage.
fixed by: on TG it is fixed by regenerative mesh, this could be tweaked to fit our ointment/silva sulfa system or these could be added. anti-infection meds are also useful in fighting these.
higher levels: more bonus damage, faster infection rates.

For more info: the TG wiki https://tgstation13.org/wiki/Guide_to_Wounds

## Why its good for the game.

Currently, internal damage is highly RNG, and it can fuck over your entire round. getting hit in the foot with a single bullet can slow you down permanently, basically leaving you for dead.

With this system, the limits of what causes wounds is less reliant on RNG.

it also focuses heavily on blood as a damage type. This means that, no matter how many healing memechems you have in you, you will still bleed to death after taking 20 bullets. multiple of these tier 2+ wounds is basically a death sentence for anyone. 

yes there are clotting medications, but these could be given major drawbacks such as, slowing the user down, confusion? maybe drowsiness. I am not sure. perhaps they could be made unique to medipens? so you cannot get an IV full of it and just tank bullets.

When you get a wound, you are on a timer, you have so long before you die. you must fix this, else you will succumb to infection/bloodloss. This, in my opinion, is much better than the current wound system. IB, you get it and you inject yourself with blood every 10 minutes else you die. broken bones you either splint, or you die from slowdown, OR its in your chest and you die of a heart attack. 

Even if you do manage to fix these in time, you will still be missing a good chunk of blood, so if you get multiple, even if you manage to fix them it might result in your death. 

Overall, this means you can have a much more back and forth fight with security, rather than it being "sec uses autorifles and hopes for a bone break"

## Feature freeze.

This would be a massive undertaking, nobody should take it up lightly, or anytime soon. I am posting this now to hear what people think.

**Comment by S34NW** [comment]
The solution to `perhaps they could be made unique to medipens? so you cannot get an IV full of it and just tank bullets.` Is simple, make too much of it be harmful.

What, however, is stopping you from just strapping a load of blood IVs to you and running about with that plugged in?

**Reply by hal9000PR** [reply]
you could make the chem `blood` not increase a persons blood if they have a slash/puncture wound.

**Reply by roarbark** [reply]
blood could be less available to the general crew by being medical ID locked and/or blood could be made to drip in slower at dangerous levels, but faster at lethal levels to negatively impact its use in combat. it could also be changed to have the chem only restore just as much blood as you're losing it while you have wounds, stuff like that

**Comment by Christasmurf** [comment]
A suggestion, would there be a way to make it if you drag someone across the floor it makes certain wounds worse (i.e. bones) or more infected (i.e. slash and burn)? 

Using roller beds or the paramedic ambulance tray would not have this issue. This could encourage more use of the paramedic and people applying first aid where they are and calling for medical as opposed to just dragging them straight to the medbay.

**Reply by hal9000PR** [reply]
Interesting suggestion.

**Comment by procdrone** [comment]
I personally am not a fan, the station combat balance was always oriented around crew laser-based weaponry, so antags don't get lasting damage so they don't require any surgery to continue - any internal damage is usually a game over for antags that don't respawn or aheal. 

While syndicate weapons are specifically ballistic, so they do inflict internal damage, so the officers/crew that engaged the antag can be removed temporarily from the vicious hunt by forcing said personnel to visit medbay for operations, giving reprieve and window of operations.

To me, it just feels like massive nerf to antagonists who already have it hard in a very PvP environment + medbay already barely handles current traffic as it is, should any heavier fighting happen.

Unless the wounds are made trivial to be removed, but then, what's the point in having them in the first place?

**Reply by Threes27** [reply]
Does this not further reinforce that paradigm? Ballistic ranged weapons have 2 serious wounds they can generate that demand detachment, whereas lasers can only cause burn wounds that multiply damage taken and add an infection timer. Ballistics still have a massive upper hand.

**Reply by hal9000PR** [reply]
First tier wounds are easy to remove, a bit of sutures, a medipen, bandages, firelock crushing yourself. They aren't hard to remove, they are just a threat that your have to deal with. Applying more damage "upgrades" this and makes it harder to remove. So getting a burn wound isnt a death sentence. But it does incentivise you to not get lasered into crit. 

Also, sec tends to jump to autorifles when they have a carbon target that they need to crack anyways. They just gotta hope for the RNG bone break on their feet! Cos a 20% chance to lose each shot is interesting and good balance.

**Comment by Threes27** [comment]
Would this include potentially adding new items, chems, or crafting recipes? Improvised splints from metal rods and cable cuffs, tourniquets from cloth and wood, TG-style chems that slow bleeding at the cost of tox/oxy/organ damage, etc. could all be interesting.

**Reply by hal9000PR** [reply]
new chems, definitely
new items, sutures, bone tape, bone gel being consumable and maybe regenerative mesh
new crafting recipes? not sure
